packages/orchestration/src/typeGuards.js

```diff
@@ -133,6 +134,7 @@ export const DenomAmountShape = { denom: DenomShape, value: M.bigint() };
 /** @type {TypedPattern<AmountArg>} */
 export const AmountArgShape = M.or(AmountShape, DenomAmountShape);
 
+/** @type {TypedPattern<RequestQuery>} */
```

Comment:
`RequestQuery['data']` is `Uint8Array`. Was going to suggest using `TypedPattern<JsonSafe<RequestQuery>>`, but we need to preserve `RequestQuery['height']: bigint`

Reply:
thanks, I'll correct this